### ISSUE TITLE
Build Error: unsupported features in 11 methods

### ISSUE THREAD

**Issue by weolopez** [quarkusio/quarkus#1960]
Please Close: this is related to local nexus.

Building native https://github.com/quarkusio/quarkus-quickstarts/tree/master/using-vertx
Using: GraalVM Version 1.0.0-rc14 

```
Error: unsupported features in 11 methods
Detailed message:
Error: Class initialization failed: io.vertx.core.json.Json
Original exception that caused the problem: java.lang.NoClassDefFoundError: com/fasterxml/jackson/core/Versioned
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:468)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at sun.misc.Unsafe.ensureClassInitialized(Native Method)
	at com.oracle.svm.hosted.ClassInitializationSupportImpl.ensureClassInitialized(ClassInitializationSupportImpl.java:199)
	at com.oracle.svm.hosted.ClassInitializationSupportImpl.computeInitKindAndMaybeInitializeClass(ClassInitializationSupportImpl.java:162)
	at com.oracle.svm.hosted.ClassInitializationSupportImpl.computeInitKindAndMaybeInitializeClass(ClassInitializationSupportImpl.java:138)
	at com.oracle.svm.hosted.ClassInitializationSupportImpl.maybeInitializeHosted(ClassInitializationSupportImpl.java:103)
	at com.oracle.svm.hosted.SVMHost.registerType(SVMHost.java:179)
	at com.oracle.graal.pointsto.meta.AnalysisUniverse.createType(AnalysisUniverse.java:263)
	at com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:204)
	at com.oracle.graal.pointsto.meta.AnalysisUniverse.lookup(AnalysisUniverse.java:181)
	at com.oracle.graal.pointsto.meta.AnalysisMethod.getDeclaringClass(AnalysisMethod.java:323)
	at com.oracle.graal.pointsto.meta.AnalysisMethod.<init>(AnalysisMethod.java:114)
	at com.oracle.graal.pointsto.meta.AnalysisUniverse.createMethod(AnalysisUniverse.java:411)
	at com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:399)
	at com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.lookupMethod(WrappedConstantPool.java:115)
	at org.graalvm.compiler.java.BytecodeParser.lookupMethod(BytecodeParser.java:4035)
	at org.graalvm.compiler.java.BytecodeParser.genInvokeStatic(BytecodeParser.java:1453)
	at org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5020)
	at org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3184)
	at org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:2993)
	at org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:891)
	at org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:785)
	at org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:95)
	at org.graalvm.compiler.phases.Phase.run(Phase.java:49)
	at org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:197)
	at org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
	at org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
	at com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:213)
	at com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:332)
	at com.oracle.graal.pointsto.flow.MethodTypeFlow.doParse(MethodTypeFlow.java:310)
	at com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureParsed(MethodTypeFlow.java:300)
	at com.oracle.graal.pointsto.flow.MethodTypeFlow.addContext(MethodTypeFlow.java:107)
	at com.oracle.graal.pointsto.DefaultAnalysisPolicy$DefaultVirtualInvokeTypeFlow.onObservedUpdate(DefaultAnalysisPolicy.java:191)
	at com.oracle.graal.pointsto.flow.TypeFlow.notifyObservers(TypeFlow.java:352)
	at com.oracle.graal.pointsto.flow.TypeFlow.update(TypeFlow.java:394)
	at com.oracle.graal.pointsto.BigBang$2.run(BigBang.java:508)
	at com.oracle.graal.pointsto.util.CompletionExecutor.lambda$execute$0(CompletionExecutor.java:169)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.core.Versioned
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 52 more
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved method during parsing: io.vertx.core.json.Json.checkAndCopy(java.lang.Object, boolean). To diagnose the issue you can use the --allow-incomplete-classpath option. The missing method is then reported at run time when it is accessed the first time.
Trace: 
	at parsing io.vertx.core.json.JsonArray.copy(JsonArray.java:579)
Call path from entry point to io.vertx.core.json.JsonArray.copy(): 
	at io.vertx.core.json.JsonArray.copy(JsonArray.java:577)
	at io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.transform(JsonArrayMessageCodec.java:39)
	at io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.transform(JsonArrayMessageCodec.java:21)
	at io.vertx.core.eventbus.impl.MessageImpl.body(MessageImpl.java:96)
	at io.vertx.axle.core.eventbus.Message.body(Message.java:108)
	at org.acme.vertx.EventResource$$Lambda$581/2123734329.apply(Unknown Source)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:602)
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:577)
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442)
	at org.xnio.nio.WorkerThread.safeRun(WorkerThread.java:612)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:479)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:473)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:193)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
--------------------------------------------------------------------------------------------
-- WARNING: The above stack trace is not a real stack trace, it is a theoretical call tree---
-- If an interface has multiple implementations SVM will just display one potential call  ---
-- path to the interface. This is often meaningless, and what you actually need to know is---
-- the path to the constructor of the object that implements this interface.              ---
-- Quarkus has attempted to generate a more meaningful call flow analysis below          ---
---------------------------------------------------------------------------------------------

Possible path to io.vertx.core.json.JsonArray.copy():i
	io.vertx.core.json.JsonArray.copy():i
	io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.transform(io.vertx.core.json.JsonArray):i
	io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.transform(io.vertx.core.json.JsonArray):i
	io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.transform(java.lang.Object):j
This is an implementation of io.vertx.core.eventbus.MessageCodec printing path to constructors of io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec



Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved method during parsing: io.vertx.core.json.Json.checkAndCopy(java.lang.Object, boolean). To diagnose the issue you can use the --allow-incomplete-classpath option. The missing method is then reported at run time when it is accessed the first time.
Trace: 
	at parsing io.vertx.core.json.JsonObject.copy(JsonObject.java:804)
Call path from entry point to io.vertx.core.json.JsonObject.copy(): 
	at io.vertx.core.json.JsonObject.copy(JsonObject.java:797)
	at io.vertx.core.eventbus.impl.codecs.JsonObjectMessageCodec.transform(JsonObjectMessageCodec.java:39)
	at io.vertx.core.eventbus.impl.codecs.JsonObjectMessageCodec.transform(JsonObjectMessageCodec.java:21)
	at io.vertx.core.eventbus.impl.MessageImpl.body(MessageImpl.java:96)
	at io.vertx.axle.core.eventbus.Message.body(Message.java:108)
	at org.acme.vertx.EventResource$$Lambda$581/2123734329.apply(Unknown Source)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:602)
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:577)
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442)
	at org.xnio.nio.WorkerThread.safeRun(WorkerThread.java:612)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:479)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:473)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:193)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
--------------------------------------------------------------------------------------------
-- WARNING: The above stack trace is not a real stack trace, it is a theoretical call tree---
-- If an interface has multiple implementations SVM will just display one potential call  ---
-- path to the interface. This is often meaningless, and what you actually need to know is---
-- the path to the constructor of the object that implements this interface.              ---
-- Quarkus has attempted to generate a more meaningful call flow analysis below          ---
---------------------------------------------------------------------------------------------

Possible path to io.vertx.core.json.JsonObject.copy():i
	io.vertx.core.json.JsonObject.copy():i
	io.vertx.core.eventbus.impl.codecs.JsonObjectMessageCodec.transform(io.vertx.core.json.JsonObject):i
	io.vertx.core.eventbus.impl.codecs.JsonObjectMessageCodec.transform(io.vertx.core.json.JsonObject):i
	io.vertx.core.eventbus.impl.codecs.JsonObjectMessageCodec.transform(java.lang.Object):j
This is an implementation of io.vertx.core.eventbus.MessageCodec printing path to constructors of io.vertx.core.eventbus.impl.codecs.JsonObjectMessageCodec


Possible path to io.vertx.core.json.JsonObject.copy():i
	io.vertx.core.json.JsonObject.copy():i
	io.vertx.core.impl.DeploymentManager.doDeploy(java.lang.String, io.vertx.core.DeploymentOptions, io.vertx.core.impl.ContextInternal, io.vertx.core.impl.ContextInternal, io.vertx.core.Handler, java.lang.ClassLoader, io.vertx.core.Verticle[]):v
	io.vertx.core.impl.DeploymentManager.lambda$doDeployVerticle$2(java.lang.String, io.vertx.core.DeploymentOptions, io.vertx.core.Handler, io.vertx.core.spi.VerticleFactory, java.lang.ClassLoader, io.vertx.core.impl.ContextInternal, io.vertx.core.impl.ContextInternal, java.util.Iterator, io.vertx.core.AsyncResult):v
	io.vertx.core.impl.DeploymentManager$$Lambda$50a490b46cb4511b376671a9a267fc9c5e596c18.handle(java.lang.Object):v
This is an implementation of io.vertx.core.Handler printing path to constructors of io.vertx.core.impl.DeploymentManager$$Lambda$50a490b46cb4511b376671a9a267fc9c5e596c18



Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved method during parsing: io.vertx.core.json.Json.decodeValue(io.vertx.core.buffer.Buffer, java.lang.Class). To diagnose the issue you can use the --allow-incomplete-classpath option. The missing method is then reported at run time when it is accessed the first time.
Trace: 
	at parsing io.vertx.core.json.JsonArray.fromBuffer(JsonArray.java:660)
Call path from entry point to io.vertx.core.json.JsonArray.fromBuffer(Buffer): 
	at io.vertx.core.json.JsonArray.fromBuffer(JsonArray.java:660)
	at io.vertx.core.json.JsonArray.<init>(JsonArray.java:72)
	at io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.decodeFromWire(JsonArrayMessageCodec.java:34)
	at io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.decodeFromWire(JsonArrayMessageCodec.java:21)
	at io.vertx.core.eventbus.impl.clustered.ClusteredMessage.decodeBody(ClusteredMessage.java:181)
	at io.vertx.core.eventbus.impl.clustered.ClusteredMessage.body(ClusteredMessage.java:88)
	at io.vertx.axle.core.eventbus.Message.body(Message.java:108)
	at org.acme.vertx.EventResource$$Lambda$581/2123734329.apply(Unknown Source)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:602)
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:577)
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442)
	at org.xnio.nio.WorkerThread.safeRun(WorkerThread.java:612)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:479)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:473)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:193)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
--------------------------------------------------------------------------------------------
-- WARNING: The above stack trace is not a real stack trace, it is a theoretical call tree---
-- If an interface has multiple implementations SVM will just display one potential call  ---
-- path to the interface. This is often meaningless, and what you actually need to know is---
-- the path to the constructor of the object that implements this interface.              ---
-- Quarkus has attempted to generate a more meaningful call flow analysis below          ---
---------------------------------------------------------------------------------------------

Possible path to io.vertx.core.json.JsonArray.fromBuffer(io.vertx.core.buffer.Buffer):v
	io.vertx.core.json.JsonArray.fromBuffer(io.vertx.core.buffer.Buffer):v
	io.vertx.core.json.JsonArray.<init>(io.vertx.core.buffer.Buffer):v
	io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.decodeFromWire(int, io.vertx.core.buffer.Buffer):i
	io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.decodeFromWire(int, io.vertx.core.buffer.Buffer):i
	io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec.decodeFromWire(int, io.vertx.core.buffer.Buffer):j
This is an implementation of io.vertx.core.eventbus.MessageCodec printing path to constructors of io.vertx.core.eventbus.impl.codecs.JsonArrayMessageCodec
```

**Comment by cescoffier** [comment]
Hum.. does not happen in the `development` branch, so probably fixed already.

**Comment by gsmet** [comment]
I think it's due to the same bootstrap issue we had.

We release 0.13.1 this evening, it will hopefully be fixed.

**Comment by gsmet** [comment]
@weolopez can you check if it's fixed with the just release 0.13.1?

**Comment by ml5174** [comment]
Looks like our corporate nexus was not updated with the BOM dependencies and it manifested itself in this way.  Thanks. 
Can close
